### ISSUE TITLE
How to use with Angular 9

### ISSUE THREAD

**Issue by ReedOwens** [MauriceConrad/Photon#42]
I'm starting an Electron project using Angular 9 as the JS Engine.  I've got other frameworks like Proton, and Xel (which uses web components) to work with Angular 9/Electron.   I'm not sure how to get this cool package of yours to work.

For Web-Components there is a special change needed in the app.modules.ts file to add CUSTOM_SCHEMA to the schemas so that they are picked up.   I've done this.

Your doc says  to use 
```javascript
const Photon = require("electron-photon");
```
This doesn't work in Angular 9.    It looks like an error trying to load the CSS.

Here is the console log:
![image](https://user-images.githubusercontent.com/18237938/78721193-0dad4000-78f5-11ea-81e4-a12905c97226.png)

**Comment by jonaaix** [comment]
Did you try `import * as Photon from 'electron-photon'` ?
As far as I know, you can not use require to handle ES6 imports.

**Comment by ReedOwens** [comment]
Yes, I did that but none of the CSS is in the app.   On the plus side, there are no errors in the log.
